app/uk/gov/hmrc/helptosavestub/controllers/UserInfoAPIController.scala

```diff
@@ -79,11 +83,50 @@ class UserInfoAPIController extends BaseController {
   }
 
 
+  //DATA FOR AIR GAP TESTING
```

Comment:
Can we put the air gap testing stuff into some object, e.g.
```scala
object AirGapTesting{
  type Token = String
  val scenario1User = ...
  val hardCodedData: Map[Token,UserInfo] = Map(...)
}
```
This way the air gap testing stuff is separated more clearly from the other stuff